ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -78,15 +78,15 @@ open class BadgeField: UIView {
         static let badgeHeight: CGFloat = 26
         static let badgeMarginHorizontal: CGFloat = 5
         static let badgeMarginVertical: CGFloat = 5
-        static let emptyTextFieldString: String = zeroWidthSpace
+        static let emptyTextFieldString: String = emptyString
         static let dragAndDropMinimumPressDuration: TimeInterval = 0.2
         static let dragAndDropScaleAnimationDuration: TimeInterval = 0.3
         static let dragAndDropScaleFactor: CGFloat = 1.10
         static let dragAndDropPositioningAnimationDuration: TimeInterval = 0.2
         static let labelMarginRight: CGFloat = 5
         static let textStyleFont: UIFont = TextStyle.subhead.font
         static let textFieldMinWidth: CGFloat = 100
-        static let zeroWidthSpace: String = "\u{200B}"
+        static let emptyString: String = ""
```

Comment:
This can be removed and you can use "" as the default value for emptyTextFieldString.

Reply:
why did the previous author use unicode for zerowidthspace? there must have been a reason?

Reply:
I couldn't find any references from blame logs to find any concrete answers. The information I was able to find online was that this character are typically used for [line wrapping](https://www.ptiglobal.com/2018/04/26/the-beauty-of-unicode-zero-width-characters/). 

In addition we use them in [AvatarView](https://github.com/microsoft/fluentui-apple/blob/4cfde746f2dd7d9b337829d6119e797063ce72ab/macos/FluentUI/AvatarView/AvatarView.swift#L349) as character separators. It seemed out of context in this case. I was hoping either of you would have more context :) 

If not, I would probably reach out to broader team to double check on this.

Reply:
the previous author aren't the the team anymore. removing this doesn't have any affect on badge field layout?

Reply:
Nope, I have attached the before/after changes pics below.

### Zero space characters

![peoplepicker_before](https://user-images.githubusercontent.com/63682282/98876778-d88d4100-2433-11eb-89de-2e5e18afb668.png) 
![badge_before](https://user-images.githubusercontent.com/63682282/98876782-d9be6e00-2433-11eb-95bc-8d7b7b961f8d.png)

### Empty string
![peoplepicker_after](https://user-images.githubusercontent.com/63682282/98876830-f5297900-2433-11eb-8e50-e949304affa6.png)
![badge_after](https://user-images.githubusercontent.com/63682282/98876833-f5c20f80-2433-11eb-8f42-520c7d386960.png)

Reply:
I posted this question to broader team across FX to check if someone else have more knowledge about the use of this context and below is the response. 

> it looks like it was there since BadgeField came in so probably just got overlooked in the initial PR. It's maybe possible removing may expose other code that is incorrectly assuming that a string length of 0 was a deletion rather than just initial state? But most likely I'd guess using "" as a drop-in replacement should be sufficient.

Reply:
ifthat's the case, i am not sure why we need zerowithspace static let at all?  just directly set emptyTextFieldString with ""?

Reply:
I have updated this.